backend/apps/slack/management/commands/slack_sync_data.py

```
@@ -41,90 +31,116 @@ def handle(self, *args, **options):
-                        member
-                        for member_data in response["members"]
-                        if (member := Member.update_data(member_data, workspace))
-                    )
-                    total_members += len(response["members"])
-
-                    cursor = response.get("response_metadata", {}).get("next_cursor")
-                    if not cursor:
-                        break
+                response = func(*args, **kwargs)
+                if not response.get("ok", False):
+                    raise SlackApiError(f"{func.__name__} failed", response)
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)